.github/workflows/frontend.yaml

```diff
@@ -0,0 +1,37 @@
+name: Frontend
```

Comment:
This file needs to be known by dev kit

Reply:
I add it to the main issue, to be considered. IMHO, until we don't have 2 bundles using webpack encore it is not necessary. Take a look at: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/.github/workflows/test-platforms.yaml. That file is also not on dev-kit, but it is too specific of the persistence bundle, so it is just fine for now.

Reply:
Important is that we generate the correct branch names when we tag 4.0 stable